packages/iis_otel/changelog.yml

```
@@ -0,0 +1,6 @@
+# newer versions go on top
+- version: "0.1.0"
+  changes:
+    - description: Initial draft of the IIS OTEL content package
```

Although we come across such descriptions, but do we actually mean them to be drafts? Changelogs usually describe what was introduced in this version, not the state of the work. “Draft” sounds like it’s unfinished, which isn’t a helpful to users reading release notes. 
```suggestion
    - description: Added IIS OpenTelemetry content package.
```
OR
```suggestion
    - description: First release of the IIS OpenTelemetry content package. 
```